android/src/main/java/com/saranshmalik/rnzendeskchat/RNZendeskChat.java

```diff
@@ -156,6 +157,7 @@ public void startChat(ReadableMap options) {
         Providers providers = Chat.INSTANCE.providers();
         setUserIdentity(options);
         setVisitorInfo(options);
+        setUserIdentity(options);
```

Comment:
what is the reason of call the same method twice?

Reply:
Probably a bad merge, thanks for the point out. Will remove this.